yocto-check-layer broken as of bbcb7e5

yocto-check-layer(-wrapper) seems to be broken now that we've merged k1 mainline support:

```
======================================================================
INFO: FAIL: test_world_inherit_class (common.CommonCheckLayer)
INFO: ----------------------------------------------------------------------
INFO: Traceback (most recent call last):
  File "/home/tgamblin/yocto/openembedded-core/scripts/lib/checklayer/cases/common.py", line 105, in test_world_inherit_class
    self.fail('\n'.join(msg))
AssertionError: Layer meta-riscv failed additional checks from yocto-check-layer.bbclass
See below log for specific recipe parsing errors:

Generating signatures failed. This might be due to some parse error and/or general layer incompatibilities.
Command: BB_ENV_PASSTHROUGH_ADDITIONS="$BB_ENV_PASSTHROUGH_ADDITIONS BB_SIGNATURE_HANDLER" BB_SIGNATURE_HANDLER="OEBasicHash" BB_ENV_PASSTHROUGH_ADDITIONS="$BB_ENV_PASSTHROUGH_ADDITIONS INHERIT" INHERIT="yocto-check-layer" bitbake -S lockedsigs world
Output:
NOTE: Reconnecting to bitbake server...
NOTE: Retrying server connection (#1)... (16:12:44.603335)
Loading cache...done.
Loaded 0 entries from dependency cache.
Parsing recipes...ERROR: /home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/boot-bundle.bb: Unable to get checksum for boot-bundle SRC_URI entry boot-bundle.its: file could not be found
The following paths were searched:
/home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/boot-bundle-1.0/nodistro/boot-bundle.its
/home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/boot-bundle/nodistro/boot-bundle.its
/home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/files/nodistro/boot-bundle.its
/home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/boot-bundle-1.0/qemux86-64/boot-bundle.its
/home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/boot-bundle/qemux86-64/boot-bundle.its
/home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/files/qemux86-64/boot-bundle.its
/home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/boot-bundle-1.0/qemuall/boot-bundle.its
/home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/boot-bundle/qemuall/boot-bundle.its
/home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/files/qemuall/boot-bundle.its
/home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/boot-bundle-1.0/x86-64/boot-bundle.its
/home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/boot-bundle/x86-64/boot-bundle.its
/home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/files/x86-64/boot-bundle.its
/home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/boot-bundle-1.0/boot-bundle.its
/home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/boot-bundle/boot-bundle.its
/home/tgamblin/yocto/meta-riscv/recipes-bsp/u-boot/files/boot-bundle.its
ERROR: Parsing halted due to errors, see error messages above

Summary: There were 2 ERROR messages, returning a non-zero exit code.

----------------------------------------------------------------------
INFO: Ran 13 tests in 29.775s
INFO: FAILED
INFO:  (failures=1, errors=3, skipped=3)
INFO:
INFO: Summary of results:
INFO:
INFO: meta-riscv ... FAIL
```

@michaelopdenacker I think there's a path problem for boot-bundle.its in boot-bundle.bb. I've bisected it to commit `bbcb7e5`.

--- a/recipes-bsp/u-boot/boot-bundle.bb
+++ b/recipes-bsp/u-boot/boot-bundle.bb
@@ -3,6 +3,8 @@ LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
 PROVIDES = "boot-bundle"
 
+COMPATIBLE_MACHINE = "(k1)"
+
 # Useful to use mainline U-Boot when it doesn't support MMC yet
 # This way, everything can be loaded from the vendors U-Boot SPL
 # when it loads mainline U-Boot.

